### PR TITLE
Trusted Cluster Serving with Occlum - Local

### PR DESCRIPTION
**Pull request by qiyuangong**
* Trusted Cluster Serving with occlum in single container.
* Add document & README

Co-authored-by: 
@CharleneHu94 huyabai <yabai.hu@intel.com>
@yuanwu2017 yuanwu <yuan.wu@intel.com>

### CODE REVIEW

File: ppml/trusted-cluster-serving/scala/docker-occlum/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    no-zlib no-async no-tests && \
+    make -j `getconf _NPROCESSORS_ONLN` && make install
+
+RUN git clone https://github.com/redis/redis.git && \
+    cd redis && \
+    git checkout 6.0.6 && \
+    make -j `getconf _NPROCESSORS_ONLN` BUILD_TLS=yes && make PREFIX=/opt/redis install
+
+# stage. 2 Flink
+# FROM occlum/occlum:0.21.0-ubuntu18.04 as flink-jobmanager
```
Comment (glorysdj) [inline ppml/trusted-cluster-serving/scala/docker-occlum/Dockerfile:49]: not From a image? so stage 2,3,4 is not really a multi-stage. it seems like multi-run in s single image.
please dont use too many RUN, try using less RUN command1 && command2

Comment (qiyuangong) [reply ppml/trusted-cluster-serving/scala/docker-occlum/Dockerfile]: Addressed. Changed to multi-stages.

File: ppml/trusted-cluster-serving/scala/docker-occlum/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+RUN cd /opt && \
+    mkdir resnet50 && \
+    cd resnet50 && \
+    wget -c "https://sourceforge.net/projects/analytics-zoo/files/analytics-zoo-models/openvino/2018_R5/resnet_v1_50.bin/download" -O resnet_v1_50.bin && \
+    wget -c "https://sourceforge.net/projects/analytics-zoo/files/analytics-zoo-models/openvino/2018_R5/resnet_v1_50.xml/download" -O resnet_v1_50.xml
+
+# TODO will be replace with release url
+RUN mkdir -p /opt/analytics-zoo && \
+    cd /opt/analytics-zoo && \
+    wget -c "https://oss.sonatype.org/content/repositories/snapshots/com/intel/analytics/zoo/analytics-zoo-bigdl_0.12.1-spark_2.4.3/0.10.0-SNAPSHOT/analytics-zoo-bigdl_0.12.1-spark_2.4.3-0.10.0-20210322.210553-65-cluster-serving-all.zip" && \
```
[the remark this answers is not in the document]
Comment (qiyuangong) [reply ppml/trusted-cluster-serving/scala/docker-occlum/Dockerfile]: Addressed. Changed to build AZ.

**Comment by qiyuangong** [comment]
Build passed in Local & server.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**